aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -142,13 +142,13 @@ FunctionalTensorWrapper::FunctionalTensorWrapper(const Tensor& view_value, const
     ),
     value_(view_value)
 {
-  set_constructor_metadata();
   // Copy the original tensor's ViewMeta vector and push the current one.
   if (base->view_metas_.size() > 0) {
       view_metas_ = base->view_metas_;  // copy
   }
   view_metas_.push_back(meta);
   storage_ = base->storage_; // alias this tensor's storage with the base tensor's
+  set_constructor_metadata();
```

Comment:
not sure why this had to move

Reply:
I can move it back - it was necessary before when I was trying to sync() before metadata calls. `set_constructor_metadata()` ended up invoking a size or stride call somewhere, which involved trying to access the storage before we properly set it.